api/cart.js

```diff
@@ -0,0 +1,47 @@
+const express = require('express');
+const cartRouter = express.Router;
+const { Cart } = require('../db/models/index');
+
+cartRouter.post("/", async (req, res, next) => {
+    try {
+        const { orderId, quantity } = req.body;
+        const newOrder = await addItemToCart ({ orderId, quantity });
+        res.send(newOrder);
+    } catch (error) {
+        console.error("Could not add item to cart")
+        throw error;
+    }
+})
+
+cartRouter.delete("/:cartId", async (req, res, next) => {
+    try {
+        const { cartId } = req.params;
+        const cart = await getOrderById(id);
+        if (req.cart === cart.cartId) {
+            const deletedCart = await destroyCart(cartId);
```

Comment:
- The same for here and for line 19.
- For line 19, be careful of the fact that getOrderById is a db method for Orders, not Cart.
```suggestion
            const deletedCart = await Cart.destroyCart(cartId);
```